tests/e2e/001-createWallets.spec.ts

```diff
@@ -8,7 +8,9 @@ const { getByText, getByLabelText, findByLabelText, findByText } = queries;
 
 const user = USER.SINGLE();
 
-const commonCreateWalletUserCreate = async () => {
+const commonCreateWalletUserCreate = async (
+  connectToLightningWallet = true
```

Comment:
Would be nice to use an `options` object here to make this clearer in situations where this is used. I.e.:

```ts
async (options?: { connectToLightningWallet?: boolean = true }) 
```


Would be clearer in the place where it's used:
```ts
await commonCreateWalletUserCreate({connectToLightningWallet: false});
```
Something like this. Does this make sense?

Reply:
Yes, good idea. Fixed